tests/pester/release.tests.ps1

```diff
@@ -0,0 +1,58 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $false)]
+  [string]
+  $versionFilePath = "./version.json"
+)
+
+Describe "version.json file tests" {
+
+  Context "version.json file tests" {
+
+    BeforeAll {
+      $versionFile = Get-Content $versionFilePath -Raw | ConvertFrom-Json
+      $gitRepoLatestTag = git describe --tags --abbrev=0
+      $releaseNotesUrlSplitLast = $versionFile.releaseNotes.Split("/")[-1]
+      $releaseNotesUrlStart = "https://github.com/Azure/bicep-lz-vending/releases/tag/v"
+
+      # Download the previous version.json file from the repo
+      $previousVersionRawUrl = "https://raw.githubusercontent.com/Azure/bicep-lz-vending/v1.2.2/version.json"
```

Comment:
The tag here is hard coded - is this intentional? We can use `$gitRepoLatestTag` instead?

Reply:
Good spot left over from testing. Fixed